types/office-js/index.d.ts

```diff
@@ -163,78 +289,101 @@ declare namespace Office {
         /**
-         * Synchronously delivers a message from the dialog to its parent add-in.
+         * Delivers a message from the dialog box to its parent/opener page. The page calling this API must be on the same domain as the parent. 
          * @param messageObject Accepts a message from the dialog to deliver to the add-in.
          */
         messageParent(messageObject: any): void;
         /**
          * Closes the UI container where the JavaScript is executing.
-         *
-         * Supported hosts: Outlook - Minimum requirement set: Mailbox 1.5
-         *
+         * 
+         * @remarks
          * The behavior of this method is specified by the following:
-         *
-         * Called from a UI-less command button: No effect. Any dialog opened by displayDialogAsync will remain open.
-         *
-         * Called from a taskpane: The taskpane will close. Any dialog opened by displayDialogAsync will also close. If the taskpane supports pinning and was pinned by the user, it will be un-pinned.
-         *
-         * Called from a module extension: No effect.
+         * 
+         * - Called from a UI-less command button: No effect. Any dialog opened by displayDialogAsync will remain open.
+         * 
+         * - Called from a taskpane: The taskpane will close. Any dialog opened by displayDialogAsync will also close. If the taskpane supports pinning and was pinned by the user, it will be un-pinned.
+         * 
+         * - Called from a module extension: No effect.
+         * 
+         * Hosts: Excel, Word, PowerPoint, Outlook (Minimum requirement set: Mailbox 1.5)
          */
         closeContainer(): void;
     }
     export interface DialogOptions {
         /**
-         * Optional. Defines the width of the dialog as a percentage of the current display. Defaults to 99%. 250px minimum.
+         * Defines the width of the dialog as a percentage of the current display. Defaults to 80%. 250px minimum.
          */
         height?: number,
         /**
-         * Optional. Defines the height of the dialog as a percentage of the current display. Defaults to 99%. 150px minimum.
+         * Defines the height of the dialog as a percentage of the current display. Defaults to 80%. 150px minimum.
          */
         width?: number,
         /**
-         * Optional. Determines whether the dialog box should be displayed within an IFrame. This setting is only applicable in Office Online clients, and is ignored on other platforms.
+         * Determines whether the dialog box should be displayed within an IFrame. This setting is only applicable in Office Online clients, and is ignored by other platforms. If false (default), the dialog will be displayed as a new browser window (pop-up). Recommended for authentication pages that cannot be displayed in an IFrame. If true, the dialog will be displayed as a floating overlay with an IFrame. This is best for user experience and performance.
```

Comment:
It's not just authentication pages.  I would put it more broadly, as pages that do not allow embedding, such as authentication pages, where the "X-Frame-Options: DENY" is present in the headers.